src/libs/actions/Report.js

```diff
@@ -1312,14 +1312,26 @@ function syncChatAndIOUReports(chatReport, iouReport) {
 }
 
 /**
- * Updates a user's notification preferences for a chat room
- *
- * @param {Number} reportID
- * @param {String} notificationPreference
+ * @param {String} reportID
```

Comment:
I think this might be a number, right?

```suggestion
 * @param {Number} reportID
```